### PR TITLE
Feed details description

### PR DESCRIPTION
**Pull request by PintoGideon**
**In this PR:**

- Added a Feed Description with an FontAwesome Icon.

**Note:**

- I added a method in the chris api's type definition file to get the feed.

### CODE REVIEW

File: src/components/feed/FeedDetails.tsx [hunk 1]
```diff
@@ -4,10 +4,41 @@ import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
...
+    const id = details.id as number;
+    this.fetchNote(id);
+  }
+
+  async fetchNote(feedId: number) {
+    const client = ChrisAPIClient.getClient();
+    const feed = await client.getFeed(feedId);
+    const note = await feed.getNote();
+    const { data } = note;
+    console.log("Fetch Note", note.data);
```
Comment (jcaianirh) [inline src/components/feed/FeedDetails.tsx]: remove console.logs from production code.

File: src/components/feed/FeedDetails.tsx [hunk 2]
```diff
@@ -24,43 +55,50 @@ class FeedDetails extends React.Component<AllProps> {
...
     }
-    return runtimeStrings.join(', ');
-
+    return runtimeStrings.join(", ");
   }
 
   render() {
     const { details } = this.props;
+    const { feedDescription } = this.state;
+    console.log("Feed Description", feedDescription);
```
Comment (jcaianirh) [inline src/components/feed/FeedDetails.tsx]: remove console.log for production code.

File: src/components/feed/FeedOutputBrowser.tsx [hunk 3]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (jcaianirh) [inline src/components/feed/FeedOutputBrowser.tsx]: @PintoGideon nothing changed in this file, i don't think you have to add this to the commit.

Comment (PintoGideon) [reply src/components/feed/FeedOutputBrowser.tsx]: Sorry @jcaianirh  That was careless from my end. Should be checked before pushing code next time.

**Pull request closed without merging**